setup.py

```diff
@@ -6,16 +6,21 @@
 import os
 
 from setuptools import find_packages, setup
+from setuptools.dist import Distribution
 
 
-libs = list(glob.glob("./bitsandbytes/libbitsandbytes*.so"))
+libs = list(glob.glob("./bitsandbytes/libbitsandbytes*.*"))
 libs = [os.path.basename(p) for p in libs]
 print("libs:", libs)
 
 
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
+# Tested with wheel v0.29.0
+class BinaryDistribution(Distribution):
+    def has_ext_modules(foo):
+        return True
```

Comment:
Should probably look into using [Hatchling](https://hatch.pypa.io/latest/) instead of any more `setup.py` shenanigans, but that's for another PR (and I'll gladly help out with that!)

Reply:
I must admit I have not changed the way I do Python wheels much in many years. This is probably very old fashioned. I would suggest following up with a modernization PR once it has been agreed if we go with cmake and what the output artifacts are